packages/studio-ui/src/web/views/FlowBuilder/sidePanelFlows/FlowNameModal.tsx

```diff
@@ -61,6 +61,8 @@ const FlowNameModal: FC<Props> = props => {
     submitText = lang.tr('rename')
   }
 
+  const isNameInvalid = name?.startsWith('skills/')
```

Comment:
This feels a little naked. Maybe there are better places for this condition to be then hardcoded in the UI. 

Then again, I don't really mind

Reply:
I thought of implementing something better, but since it's the only case we have to handle, i prefer to not spend more time. Thought of an array of illegal names, regex, etc, but didnt wanna overengineer for this specific case

Reply:
makes sense